iOS/FBNotifications/FBNotifications/Internal/Models/Components/FBNCardTextContent.m

```diff
@@ -62,7 +62,7 @@ - (instancetype)initFromDictionary:(NSDictionary<NSString *, id> *)dictionary {
 }
 
 + (nullable instancetype)contentFromDictionary:(nullable NSDictionary<NSString *, id> *)dictionary {
-    if (!dictionary) {
+    if (!dictionary || [dictionary isEqual:[NSNull null]]) {
```

Comment:
Since `NSNull.null` is a singleton, it's absolutely valid to simply check for pointer equality here simply.
What you basically want is
```objc
if (!dictionary || dictionary == NSNull.null)
```